src/migrations/1630744507207-RemoveDuplicates.ts

```diff
@@ -0,0 +1,45 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+const deleteDuplicateOrgs = async (queryRunner: QueryRunner) => {
+  await queryRunner.query(
+    `DELETE FROM organization org
+            WHERE  EXISTS (
+            SELECT FROM organization
+            WHERE  namespace = org.namespace
+            AND    id < org.id
+        );`,
+  );
+};
+
+const deleteDuplicateApps = async (queryRunner: QueryRunner) => {
+  await queryRunner.query(
+    `DELETE FROM application app
+            WHERE  EXISTS (
+            SELECT FROM application
+            WHERE  namespace = app.namespace
+            AND    id < app.id
+        );`,
+  );
+};
```

Comment:
Duplicates Apps found in the database all turned out to have the same `parentOrgId` of the first OrgId created before other duplicate orgs hence it makes sense for us to remove duplicates except the first one created.